hack/verify-gotest.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# CI script to run go vet over our code
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+# cd to the root path
+cd_root_path
+
+# run go test
+export GO111MODULE=on
+go test -v ./actuators/...
```

Comment:
Is this assuming vendor gets checked in? I think `go test ./...` should be sufficient here, unless there is some reason the vendor dir will exist?

Same is true of most of these scripts

Reply:
ok, if a folder that should not be checked is added at some point it can be excluded.